python/lsst/meas/astrom/pessimistic_pattern_matcher_b_3D.py

```diff
@@ -1204,6 +1192,119 @@ def _test_rotation_agreement(self, rot_vects, max_dist_rad):
                 tot_consent += 1
         return tot_consent
 
+    def _final_verify(self,
+                      source_array,
+                      shift_rot_matrix,
+                      max_dist_rad,
+                      min_matches):
+        """Match the all sources into the reference cat using the shift/rot
+        matrix.
+
+        After the initial set of matches and found to have enough matches,
+        refit the shift/rot matrix for a more stable solution.
```

Comment:
"and found" probably isn't want you meant, but I'm not sure what you wanted. I don't know how to interpret the imperative phrasing; am I, the user, supposed to do the refitting or is that what this function does? This could use a clear statement of what the output is.

Reply:
Fixed